content/reference/user/reports/index.md

```diff
@@ -39,50 +61,25 @@ You can change the time range with control buttons or by selecting a region on t
 
 {{< img src="timeline.png" alt="Timeline" >}}
 
-### Assertions
-
-The label below is used to display the status of the simulation (Ongoing, successful, timeout...).
-If your simulation has assertions, this label will be clickable to show the assertions results.
-You can comment the run run by clicking on the {{< icon comment-alt >}} icon.
-
-{{< img src="timeline-assertions.png" alt="Timeline assertions" >}}
-
 ### Tabs
 
 Below the navigator chart, there are tabs to switch charts.
 Each tab has the same structure except the summary that is available only for requests and groups tabs.
 
 {{< img src="tabs.png" alt="Tabs" >}}
 
-### Run Bar
+### Filter Bar {#filter-bar}
 
-This bar is a combination of buttons:
+This filter bar is a combination of buttons:
 
-- **Start / Abort**: Use this button to start a new run of the simulation, or stop the ongoing run (not available if you have a Viewer permission)
-- **Grafana**: Link to the Grafana dashboard if you have filled in the configuration in frontline.conf
-- **Generate public link**: To create a public link
 - **Switch to Summary**: Switch to [summary]({{< ref "../reports#summary" >}}) view for Requests & Groups tabs
 - buttons to filter the metrics drawn in the charts area
 
-{{< img src="run-bar.png" alt="Run bar" >}}
-
-### Generate Public Links
-
-A public link is a link of the current reports which will be accessible to anyone, without having to log-in to FrontLine. To generate a public link, click on the *Generate public link* button and choose the expiration date of your link.
-
-{{< img src="generate-public-link.png" alt="Generate public links" >}}
-
-The maximum allowed lifetime for a public link is 1 year.
-
-Once you have chosen an expiration date, you can proceed by clicking on the generate button.
-
-{{< img src="successful-generation-public-link.png" alt="Successful public link generation" >}}
-
-You can copy the public link to share your reports to non-FrontLine users, or click on the "Go" Button to access it yourself. You can click on the "OK" button to close this modal.
+{{< img src="filter-bar.png" alt="Filter bar" >}}
 
 ### Charts
 
-Each charts in FrontLine are connected to each other, so if you select a time window on a chart it will automatically change it
+Each charts in Gatling Enterprise are connected to each other, so if you select a time window on a chart it will automatically change it
```

Comment:
```suggestion
Each chart in Gatling Enterprise is connected to each other, so if you select a time window on a chart it will automatically change it
```

Reply:
```suggestion
All charts in Gatling Enterprise are connected to each other, so if you select a time window on a chart it will automatically change it
```